src/Workspaces/Core/Portable/PublicAPI.Shipped.txt

```diff
@@ -936,6 +964,7 @@ abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.MemberAccessExpression(M
 abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.MethodDeclaration(string name, System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.SyntaxNode> parameters = null, System.Collections.Generic.IEnumerable<string> typeParameters = null, Microsoft.CodeAnalysis.SyntaxNode returnType = null, Microsoft.CodeAnalysis.Accessibility accessibility = Microsoft.CodeAnalysis.Accessibility.NotApplicable, Microsoft.CodeAnalysis.Editing.DeclarationModifiers modifiers = default(Microsoft.CodeAnalysis.Editing.DeclarationModifiers), System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.SyntaxNode> statements = null) -> Microsoft.CodeAnalysis.SyntaxNode
 abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.ModuloExpression(Microsoft.CodeAnalysis.SyntaxNode left, Microsoft.CodeAnalysis.SyntaxNode right) -> Microsoft.CodeAnalysis.SyntaxNode
 abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.MultiplyExpression(Microsoft.CodeAnalysis.SyntaxNode left, Microsoft.CodeAnalysis.SyntaxNode right) -> Microsoft.CodeAnalysis.SyntaxNode
+abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.NameOfExpression(Microsoft.CodeAnalysis.SyntaxNode expression) -> Microsoft.CodeAnalysis.SyntaxNode
```

Comment:
@Pilchie this is what I ended up doing -

1. Created and switched to a new branch under `stabilization` at commit 0ecf683 (which corresponds to `v1.2.1` above)
2. Committed the changes (i.e. moved all lines appearing in PublicAPI.Unshipped.txt files to corresponding PublicAPI.Shipped.txt files (except for `IOperation` and VB scripting APIs) and re-sorted the files)
3. Switched to `stabilization` branch and synced latest
4. Merged the branch created in 1 above into `stabilization`
5. This resulted in a single merge conflict in `src/Workspaces/Core/Portable/PublicAPI.Shipped.txt` for `Microsoft.CodeAnalysis.Editing.SyntaxGenerator.NameOfExpression()` API above - I resolved the conflict by leaving this API in the unshipped file
6. Committed the merge and submitted PR #10635 for `stabilization`

Let me know in case you spot anything else that looks fishy :)